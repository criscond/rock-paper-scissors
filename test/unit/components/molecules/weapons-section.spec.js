import weaponsSection from '../../../../src/js/components/molecules/weapons-section';
describe('Weapons Section', () => {

	it('should create a section', () => {
		expect(weaponsSection()).toMatchSnapshot();
	});
});
