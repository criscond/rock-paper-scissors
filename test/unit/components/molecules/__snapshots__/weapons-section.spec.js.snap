// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weapons Section should create a section 1`] = `
<section
  class="column"
>
  <h2
    class=""
  >
    Choose your weapon
  </h2>
  <div
    class="weapons"
    role="listbox"
  >
    <div
      class="scissors"
      role="option"
    >
      <h3
        class=""
      >
        Scissors
      </h3>
    </div>
    <div
      class="paper"
      role="option"
    >
      <h3
        class=""
      >
        Paper
      </h3>
    </div>
    <div
      class="rock"
      role="option"
    >
      <h3
        class=""
      >
        Rock
      </h3>
    </div>
  </div>
</section>
`;
